src/Plutus/Types/CurrencySymbol.purs

```diff
@@ -10,28 +10,17 @@ module Plutus.Types.CurrencySymbol
 
 import Prelude
 
-import Aeson
-  ( class DecodeAeson
-  , class EncodeAeson
-  , caseAesonObject
-  , decodeAeson
-  , encodeAeson'
-  , getField
-  , JsonDecodeError(TypeMismatch)
-  )
+import Aeson (class DecodeAeson, class EncodeAeson, caseAesonObject, decodeAeson, encodeAeson', getField, JsonDecodeError(TypeMismatch))
```

Comment:
Please `make format`

Reply:
It's already formated, maybe this is from a previous commit?